### PR TITLE
Regroup and rename APIs in Sample App

### PR DESCRIPTION
**Pull request by alandn-circle**
We want to regroup API endpoints based on which API(s) they belong to (Payments, Payouts, Digital Dollar Accounts, Marketplace). We also want to add a new section entitled "Core Functionality" that contains endpoints present in **all** APIs.

The bulk of these changes lie in the navigation and the API viewers for such APIs.

### CODE REVIEW

**Comment by alandn-circle** [comment]
Once these changes are good to go, I'll also be removing the files `debug/wallets/index.vue` and `debug/businessAccount/index.vue` prior to merging to master.

**Review by MasterXen — APPROVED** [review]
Verified locally (including marketplaces) and compared to the ordering of the updated API docs

**Comment by kristinfritsch** [comment]
Some cleanup comments, but not a blocker:

- `(Caution: When using a production api key it will charge your card)` can be removed from `/debug/payouts`
- let's remove the page `/debug/` (it is linked from the home page and we should remove the link there too)
- `This sample web application demonstrates some of the features of the Circle Payments API` let's change that to `This sample web application demonstrates some of the features of the Circle APIs` with link to https://www.circle.com/en/developers

File: lib/businessAccount/depositsApi.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+function getDeposits(
+  type: string,
+  from: string,
+  to: string,
+  pageBefore: string,
+  pageAfter: string,
+  pageSize: string
+) {
+  const queryParams = {
+    destination: nullIfEmpty(type),
```
Comment (alandn-circle) [inline lib/businessAccount/depositsApi.ts:62]: forgot to change this, fixed in next commit

File: lib/businessAccount/depositsApi.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+) {
+  const queryParams = {
+    type: nullIfEmpty(type),
+    from: nullIfEmpty(from),
+    to: nullIfEmpty(to),
+    pageBefore: nullIfEmpty(pageBefore),
+    pageAfter: nullIfEmpty(pageAfter),
+    pageSize: nullIfEmpty(pageSize),
+  }
+  const url = '/v1/businessAccount/deposit'
```
Comment (MasterXen) [inline lib/businessAccount/depositsApi.ts:69]: 🐛 Endpoint is /v1/businessAccount/deposit**s**

**Review by MasterXen — APPROVED** [review]

**Comment by MasterXen** [comment]
Build failed due to:
```
/opt/circle/payments-sample-app/src/lib/businessAccount/depositsApi.ts
--
382 | 38:1  error  Delete `⏎`                          prettier/prettier
383 | 38:1  error  More than 1 blank line not allowed  no-multiple-empty-lines
```

Prettier is not playing games!

**Comment by MasterXen** [comment]
I "fixed" the extra line break violation

**Review by MasterXen — APPROVED** [review]

**Pull request merged**